APIBacklog/documentation/SupportingDocuments/API proposals/API_Proposal_Device_Location_Retrieval.md

```diff
@@ -0,0 +1,14 @@
+Age Verification API submission
+
+| **Field** | Description |
+| ---- | ----- |
+| API name | Location Retrieval |
+| API owner | Ericsson DT Orange Telefonica Vodafone |
+| Initial API Contributors | Joachim Dahlgren Ericsson & Ludovic Robert - Orange | 
+| API summary | This API allows for the API Consumer to get the raw location of a mobile device. This would serve use cases in which the device does not have built in GNSS support. Extended with subscription support it would also serve use cases for tracking such devices. Location data should be provided in form of geographical coordinates (x, y, z). The reason for this is that network identities such as CellID etc. are typically not known by developers (and should perhaps not even be exposed externally). The location could be given as a point or a shape with uncertainty area. Location is in most jurisdictions considered to be sensitive data and thereby consent by device owner/user must be verified before providing it to the developer.|

```

Comment:
z? I doubt mobile networks will ever give good height estimates, so I would limit the scope to `(x,y)`.

Reply:
@eric-murray Correct. I copy/paste the initial issue and since then we discussed this point and z should b removed. I fix this. Thanks for catching this.

Reply:
We had a [discussion](https://github.com/camaraproject/DeviceLocation/issues/94) about this in DeviceLocation WG. Apparently [3GPP TS 29.572](https://www.3gpp.org/ftp/Specs/archive/29_series/29.572/OpenAPI/2018-12/TS29572_Nlmf_Location.yaml) considers altitudes in some cases, but we had doubts about the support for this feature realistically. So far we have limited the scope of the response in v0.1 to just Circles or Polygons (not even points). Regarding the wording here for administrative purposes, maybe we don't  have to give too much details.